pkg/actuators/awsplacementgroup/controller_test.go

```diff
@@ -0,0 +1,46 @@
+package awsplacementgroup
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+var _ = Describe("AWSPlacementGroupReconciler", func() {
+	namespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{GenerateName: "mhc-test-"}}
+	var c client.Client
+	var stopMgr context.CancelFunc
+	var fakeRecorder *record.FakeRecorder
+
+	BeforeEach(func() {
+		mgr, err := manager.New(cfg, manager.Options{MetricsBindAddress: "0", Namespace: namespace.Name})
+		Expect(err).ToNot(HaveOccurred())
+
+		r := Reconciler{
+			Client: mgr.GetClient(),
+			Log:    log.Log,
+		}
+		Expect(r.SetupWithManager(mgr, controller.Options{})).To(Succeed())
+
+		fakeRecorder = record.NewFakeRecorder(1)
+		r.recorder = fakeRecorder
+
+		c = mgr.GetClient()
```

Comment:
Rather than using the mgr client (which is cached), it might be worth using a direct client which you can set up in suite_test.go